hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -358,17 +379,21 @@ class AzkabanHelper {
       logger.lifecycle("Azkaban URL: ${azkProject.azkabanUrl}");
       logger.lifecycle("Azkaban User Name: ${azkProject.azkabanUserName}\n");
```

Comment:
Please remove \n here to avoid logging password without an empty line before it.